pkg/endpoint/bpf.go

```diff
@@ -1228,13 +1269,44 @@ func (e *Endpoint) applyPolicyMapChanges() error {
 	return nil
 }
 
+// endpointPolicyLockdown puts the endpoint policy map into a lockdown
+// mode. A new bpf policy map is populated with deny all traffic entries,
+// the old map is unpinned, and the new map is pinned.
+func (e *Endpoint) endpointPolicyLockdown() error {
```

Comment:
This is a really though situation to be. This means that if we add an allow all policy that selects all identities, this will break the connectivity for the pods selected by the policy.

Since we are creating bpf maps, any can't we create a map with a bigger size and then swap the maps?

Reply:
I think that this can end up being a resource usage question for the user: Is it OK to consume more resources? If resizing the map fails, then we may still need some sort of lockdown option like this. Could the alternative with resizing the map potentially trigger an OOM and hence limit the agent's ability to mitigate issues in this scenario?